Source/WebKit/GPUProcess/media/RemoteTextTrackProxy.h

```diff
@@ -93,6 +93,7 @@ class RemoteTextTrackProxy final
     Ref<WebCore::InbandTextTrackPrivate> m_trackPrivate;
     WebCore::TrackID m_id;
     WebCore::MediaPlayerIdentifier m_mediaPlayerIdentifier;
+    size_t m_clientId { 0 };
```

Comment:
ditto.

Reply:
that's different though, this is the registrationID we received when we added ourselves to the TrackPrivateBase as client, and is needed to be removed. I'll change it to m_clientRegistrationId then